testing/cloudbuild-templates/cloud-run-template.cloudbuild.yaml

```diff
@@ -0,0 +1,69 @@
+steps:
+
+- id: 'Build Container Image'
+  name: 'gcr.io/cloud-builders/docker:latest'
+  args: ['build', '-t', 'gcr.io/${PROJECT_ID}/${_SERVICE}:${COMMIT_SHA}', '$_SAMPLE']
+
+- id: 'Push Container Image'
+  name: 'gcr.io/cloud-builders/docker:latest'
+  args: ['push', 'gcr.io/${PROJECT_ID}/${_SERVICE}:${COMMIT_SHA}']
+
+- id: 'Deploy to Cloud Run'
+  name: 'gcr.io/cloud-builders/gcloud:$_CLOUDSDK_VERSION'
+  args:
+  - 'run'
+  - 'deploy'
+  - '${_SERVICE}-${COMMIT_SHA}'
+  - '--image'
+  - 'gcr.io/${PROJECT_ID}/${_SERVICE}:${COMMIT_SHA}'
+  - '--allow-unauthenticated' # TODO: Evaluate if sample requires authenticated requests
```

Comment:
We should default to `--no-allow-unauthenticated` and make authenticated requests in the tests. If the sample must have unauthenticated tests we might relax case-by-case.

Certain org policies can prevent making the service public.